src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -113,6 +114,7 @@
     + "    col_index INTEGER NOT NULL,\n"
     + "    col_name TEXT,\n"
     + "    col_type TEXT NOT NULL,\n"
+    + "    is_indexed INTEGER NOT NULL, \n" 
```

Comment:
They could also be quite useful for joins and aggregates that are pushed
into Postgres...

On Tue, Jul 12, 2016 at 2:08 PM, jortiz16 <notifications@github.com> wrote:

> In src/edu/washington/escience/myria/coordinator/MasterCatalog.java
> <https://github.com/uwescience/myria/pull/830#discussion_r70522855>:
>
> > @@ -113,6 +114,7 @@
> >      + "    col_index INTEGER NOT NULL,\n"
> >      + "    col_name TEXT,\n"
> >      + "    col_type TEXT NOT NULL,\n"
> > +    + "    is_indexed INTEGER NOT NULL, \n"
>
> I can't think of specific cases where this might be useful. I forgot about
> lack of support for ORDER BY. In the past, creating indexes was useful when
> pushing down selection operations to Postgres. For example, with the cosmo
> data, I would need to find particles that belonged to a particular galaxy
> (grpId). For that scenario, it was useful to have an index for the grpID
> column, but if I were to run that today, I would also need enable pushing
> down the computation to postgres.
>
> Long story short, I don't think there is a strong case for index ordering.
> It seems like indexes will be useful only for selections that are pushed
> down to postgres....
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/uwescience/myria/pull/830/files/4f8f7d56457c12dc57f9f0975fc979c49d08ae0e#r70522855>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAIBYQ6ga4KctReVH12VBXFUVMISzXLVks5qVAJjgaJpZM4If5_N>
> .
>